content/syllabuses/java.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Java Basics 
+---
+
+- Java Collections
+- {{% contentlink "topics/java-specific/collections-and-datastructures" %}}
+- {{% contentlink "projects/java-specific/collections" %}}
+- Java Data Structures
+- {{% contentlink "topics/java-specific/collections-and-datastructures" %}}
+- {{% contentlink "projects/java-specific/data-structures" %}}
+- Java Generics
+- {{% contentlink "topics/java-specific/generics" %}}
+- {{% contentlink "projects/java-specific/generics" %}}
+- Spring Infrastructure 
+- {{% contentlink "topics/java-specific/introduction-to-spring-infrastructure/part-1" %}}
+- {{% contentlink "topics/java-specific/introduction-to-spring-infrastructure/part-2" %}}
+- {{% contentlink "topics/java-specific/introduction-to-spring-infrastructure/part-3" %}}
+- {{% contentlink "projects/java-specific/introduction-to-spring-infrastructure/part-1" %}}
+- {{% contentlink "projects/java-specific/introduction-to-spring-infrastructure/part-2" %}}
+- {{% contentlink "projects/java-specific/introduction-to-spring-infrastructure/part-3" %}}
```

Comment:
doesn't part 3 rely on part 2? And doesn't part 2 rely on part 1?  What is the goal that the recruits are aiming for? Should all this stuff have a card that is highlighted?

Reply:
Implemented.